src/Input/ProximityPrompt.lua

```diff
@@ -21,13 +21,13 @@ function ProximityPrompt.new(Input: Unified3DInputTypes.Unified3DInput): Unified
         Input = Input,
         Enabled = false,
         Properties = {},
-    }, ProximityPrompt)
+    }, ProximityPrompt) :: any
```

Comment:
Same as the other comment, `Unified3DInputTypes.Input` should be used instead of `any`.